src/connection.h

```diff
@@ -69,6 +69,9 @@ struct conn_info {
     long long completed_commands;
 
     int8_t status;
+
+    // slow log, only for server connection in worker thread
+    uint32_t *counts;
```

Comment:
`counts` is not very readable. How about `slow_counts` or `slow_cmd_counts` or anything that is more specific?